script/tool/lib/src/update_excerpts_command.dart

```diff
@@ -165,7 +178,9 @@ class UpdateExcerptsCommand extends PackageLoopingCommand {
               errors.add(
                   '${file.path}:$lineNumber: code block was followed by a space character instead of the language (expected "$language")');
               mode = _ExcerptParseMode.injecting;
-            } else if (line != '```$language' && line != '```rfwtxt' && line != '```json') {
+            } else if (line != '```$language' &&
+                line != '```rfwtxt' &&
+                line != '```json') {
```

Comment:
It turns out we're not enforcing formatting for the `script/tool/` directory; I'll file an issue to fix that.